### PR TITLE
NOTIF-399 Add description for each email recipient type

### PR DESCRIPTION
**Pull request by aferd**

### CODE REVIEW

File: src/types/Recipient.ts [hunk 1]
```diff
@@ -36,19 +36,24 @@ export class NotificationRecipient extends Recipient {
...
     readonly integrationId: UUID | undefined;
     readonly key: string;
+    readonly description: string | undefined;
 
     public constructor(integrationId: UUID | undefined, sendToAdmin: boolean) {
         let displayName = 'Users:';
+        let description = '';
         if (sendToAdmin) {
             displayName += ' Admins';
+            description += ' Organization administrators for your account';
```
Comment (josejulio) [inline src/types/Recipient.ts]: No need to concatenate in this case 
```suggestion
            description = 'Organization administrators for your account';
```

File: src/types/Recipient.ts [hunk 2]
```diff
@@ -36,19 +36,24 @@ export class NotificationRecipient extends Recipient {
...
         } else {
             displayName += ' All';
+            description += ' All users in your organization who subscribed to this email in their User Preferences';
```
Comment (josejulio) [inline src/types/Recipient.ts]: or this one
```suggestion
            description = 'All users in your organization who subscribed to this email in their User Preferences';
```

**Review by josejulio — CHANGES_REQUESTED** [review]
Could you add a screenshot? thanks!

**Comment by aferd** [comment]
https://user-images.githubusercontent.com/86322239/148782712-efb7746d-4a92-41ed-875d-bff73ff5173b.mov

**Comment by aferd** [comment]
Screen Shot 2022-01-10 at 9 32 47 AM

**Review by josejulio — APPROVED** [review]

**Pull request merged**